src/env/mod.rs

```diff
@@ -185,6 +185,10 @@ mod test {
             ("RPC_PROXY_RATE_LIMITING_MAX_TOKENS", "100"),
             ("RPC_PROXY_RATE_LIMITING_REFILL_INTERVAL_SEC", "1"),
             ("RPC_PROXY_RATE_LIMITING_REFILL_RATE", "10"),
+            (
+                "RPC_PROXY_RATE_LIMITING_IP_WHITE_LIST",
```

Comment:
Whitelist is one word

```suggestion
                "RPC_PROXY_RATE_LIMITING_IP_WHITELIST",
```

Reply:
Fixed, thanks!